### PR TITLE
update sv4d sampling script and readme

### PR DESCRIPTION
**Pull request by chunhanyao-stable**

### CODE REVIEW

File: .vscode/launch.json
Comment (voletiv) [inline .vscode/launch.json:1]: Remove this file

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@
 - We are releasing **[Stable Video 4D (SV4D)](https://huggingface.co/stabilityai/sv4d)**, a video-to-4D diffusion model for novel-view video synthesis. For research purposes:
     - **SV4D** was trained to generate 40 frames (5 video frames x 8 camera views) at 576x576 resolution, given 5 context frames (the input video), and 8 reference views (synthesised from the first frame of the input video, using a multi-view diffusion model like SV3D) of the same size, ideally white-background images with one object.
     - To generate longer novel-view videos (21 frames), we propose a novel sampling method using SV4D, by first sampling 5 anchor frames and then densely sampling the remaining frames while maintaining temporal consistency.
-    - Please check our [project page](https://sv4d.github.io), [tech report](https://sv4d.github.io/static/sv4d_technical_report.pdf) and [video summary](https://www.youtube.com/watch?v=RBP8vdAWTgk) for more details.
+    - Please check our [project page](), [tech report]() and [video summary]() for more details.
```
Comment (voletiv) [inline README.md:12]: Is this removal of links on purpose?

File: sgm/modules/spacetime_attention.py [hunk 2]
```diff
@@ -593,4 +593,4 @@ def forward(
         if not self.use_linear:
             x = self.proj_out(x)
         out = x + x_in
-        return out
+        return out
```
Comment (voletiv) [inline sgm/modules/spacetime_attention.py:596]: Unnecessary, please remove

**Review by voletiv — CHANGES_REQUESTED** [review]

**Comment by chunhanyao-stable** [comment]
@voletiv I resolved all the comments. Please check again. Thanks!

**Review by voletiv — APPROVED** [review]

**Pull request merged**